VPXDecoder.cpp

```diff
@@ -95,6 +100,7 @@ VPXDecoder::IMAGE_ERROR VPXDecoder::getImage(Image &image)
 
 				image.w = img->d_w;
 				image.h = img->d_h;
+				image.cs = m_last_space;
```

Comment:
What about `image.cs = img->cs;` (without `m_last_space`)?

Reply:
The problem with that is that `m_last_space` is necessary because img->cs is the correct colour space for one frame, and then UNKNOWN for the rest. I have a WEBM file 'vp9-bgrp' that causes this on the Godot-side PR.

Reply:
ok, thx for info